pages/api/backdrop.md

```diff
@@ -23,6 +23,8 @@ import Backdrop from '@material-ui/core/Backdrop';
 | <span class="prop-name required">open *</span> | <span class="prop-type">bool</span> |   | If `true`, the backdrop is open. |
 | <span class="prop-name">transitionDuration</span> | <span class="prop-type">union:&nbsp;number&nbsp;&#124;<br>&nbsp;{ enter?: number, exit?: number }<br></span> |   | The duration for the transition, in milliseconds. You may specify a single timeout for all transitions, or individually with an object. |
 
+The `ref` is **not** forwarded to the root element.
```

Comment:
Unless I'm missing something, it's forwarded to the root element (the top element in the DOM structure). Is this a matter of updating the tests?

Reply:
The root element is not always the top element. This was established in #14958

Apart from that it's just a matter of updating the tests. #15131 will clear things up.

Reply:
Should we document somewhere what's a "root element" is?

Reply:
Yes, the root element is not always the top element. Sometimes, they are multiple top elements (tooltip).
What should be the expected behavior?

Reply:
That's something I'm trying to figure out. There are still plenty of components that don't really follow the "root component" pattern. Once the coverage is high enough in #14958 we can explicitly document it. Right now the term "root element" is only informal in the docs. I'd hope we can get every component to follow 
https://github.com/mui-org/material-ui/blob/71234113e3d14bcbfe0d3acec3023d219615d74e/packages/material-ui/src/test-utils/describeConformance.js#L7-L11

Reply:
This unilateral redefinition of 'root element' is becoming more troubling. The root element has always referred to the outermost element of a component. If you don't like that for some reason, then please open a separate issue and propose a concrete alternative naming scheme, explaining the rational. Until then, the root element is the root element.

> This was established in #14958

I don't see any relevant discussion there.

Reply:
>  The root element has always referred to the outermost element of a component.

That's not true though. I inferred this from how it was handled and since #14958 was approved I don't understand why this is an issue all of the sudden. I will open an issue explaining this in more detail.

Reply:
> That's not true though.

You keep saying it's not true without providing examples. Why is this not true? I don't see the relation with #14958. This pull request improves the test coverage.

Reply:
Continued in #15138.

Reply:
Updated with cd7612d392bb030db96fe5c4966b6661c3344698 which was possible after including #15131.